samples/client/echo_api/go/model_tag.go

```diff
@@ -124,6 +126,21 @@ func (o Tag) ToMap() (map[string]interface{}, error) {
 	return toSerialize, nil
 }
 
+func (o *Tag) UnmarshalJSON(bytes []byte) (err error) {
```

Comment:
CI passes now, but models that have no required properties and no additional properties gain an unnecessary `UnmarshalJSON` function, so I'm working on updating the template to only generate the `UnmarshalJSON` when it's needed.